Library/Homebrew/cask/cmd.rb

```diff
@@ -54,35 +54,26 @@ class Cmd
     }.freeze
 
     DEPRECATED_COMMANDS = {
-      Cmd::Cache     => "brew --cache --cask",
+      Cmd::Cache     => "brew --cache [--cask]",
+      Cmd::Audit     => "brew audit [--cask]",
+      Cmd::Cat       => "brew cat [--cask]",
+      Cmd::Create    => "brew create --cask --set-name <name> <url>",
       Cmd::Doctor    => "brew doctor --verbose",
+      Cmd::Edit      => "brew edit [--cask]",
+      Cmd::Fetch     => "brew fetch [--cask]",
+      Cmd::Help      => "brew help",
       Cmd::Home      => "brew home",
-      Cmd::List      => "brew list --cask",
-      Cmd::Outdated  => "brew outdated --cask",
-      Cmd::Reinstall => "brew reinstall",
-      Cmd::Upgrade   => "brew upgrade --cask",
+      Cmd::Info      => "brew info [--cask]",
+      Cmd::Install   => "brew install [--cask]",
+      Cmd::List      => "brew list [--cask]",
+      Cmd::Outdated  => "brew outdated [--cask]",
+      Cmd::Reinstall => "brew reinstall [--cask]",
+      Cmd::Style     => "brew style",
+      Cmd::Uninstall => "brew uninstall [--cask]",
+      Cmd::Upgrade   => "brew upgrade [--cask]",
+      Cmd::Zap       => "brew upgrade --zap [--cask]",
```

Comment:
This should be `brew uninstall --zap [--cask]` right? There's no mention of `--zap` in `brew upgrade --help`.

Reply:
Yes, good catch, will fix.

Reply:
https://github.com/Homebrew/brew/pull/9298